lib/BubbleChart.js

```diff
@@ -13,13 +13,13 @@ const iface = {
     ...ChartBase.propTypes,
 
     data: PropTypes.shape({
-      datasets: PropTypes.arrayOf(PropTypes.shape({
+      dataSets: PropTypes.arrayOf(PropTypes.shape({
         values: PropTypes.arrayOf(PropTypes.shape({
           x: PropTypes.number,
           y: PropTypes.number.isRequired,
           size: PropTypes.number.isRequired,
+          marker:PropTypes.string,
```

Comment:
why did you add marker here? I do not see it being used when creating entries. Am I missing something?